hack/run-db-migrations.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/bash
 
-if [ "$WORKER_ADMINISTRATION_REGION" != "ingestion" ]; then
-    echo "Only ingestion workers can run the DB migrations, not \"$WORKER_ADMINISTRATION_REGION\" workers. Skipping..."
+if [ -n "$WORKER_RUN_DB_MIGRATIONS" ]; then
```

Comment:
Shouldn't this be `[ -z "$WORKER_RUN_DB_MIGRATIONS" ]` ?  (since the message says 'WORKER_RUN_DB_MIGRATIONS was not set')